File: PHP/01_ola_mundo.php
<?php
/**
 * 01 - Olá Mundo
 * Conceitos: echo, variáveis, concatenação, entrada do usuário
 */

// 1. O clássico
echo "Olá, Mundo!\n";

// 2. Variáveis
$nome = "Estudante";
$idade = 25;
echo "Olá, $nome! Você tem $idade anos.\n";

// 3. Entrada do usuário (via terminal)
echo "Digite seu nome: ";
$nomeUsuario = trim(fgets(STDIN));
echo "Bem-vindo(a), $nomeUsuario!\n";

// 4. Operação simples
echo "Em que ano você nasceu? ";
$anoNascimento = (int) trim(fgets(STDIN));
$anoAtual = 2026;
$idadeUsuario = $anoAtual - $anoNascimento;
echo "Você tem aproximadamente $idadeUsuario anos.\n";

File: Python/01_ola_mundo.py
"""
01 - Olá Mundo
Conceitos: print, variáveis, input, f-strings
"""

# 1. Olá Mundo
print("Olá, Mundo!")

# 2. Variáveis
nome = "Estudante"
idade = 25
print(f"Olá, {nome}! Você tem {idade} anos.")

# 3. Entrada do usuário
nome_usuario = input("Digite seu nome: ")
print(f"Bem-vindo(a), {nome_usuario}!")

# 4. Operações simples com variáveis
ano_atual = 2026
ano_nascimento = int(input("Em que ano você nasceu? "))
idade_usuario = ano_atual - ano_nascimento
print(f"Você tem aproximadamente {idade_usuario} anos.")